pkgs/by-name/br/brygada-1918/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "brygada-1918";
+  version = "3.006";
+
+  src = fetchFromGitHub {
+    owner = "kosmynkab";
+    repo = "Brygada-1918";
+    rev = "8325dc36ca87b8c7b8909c3e048fe90fd7e46c4b";
+    hash = "sha256-ePehnBYFlm+iExf2Sy25PsIWvtHlys2hcCZ4cHT+H1k=";
+  };
+
```

```suggestion

  dontConfigure = true;
  dontBuild = true;

```
This doesn't actually affect the output, but it can help speed up the build a little bit